### ISSUE TITLE
Need tutorial for preprocessing steps

### ISSUE THREAD

**Issue by alcantarar** [alcantarar/Recurrent_GRF_Prediction#4]
Users interested in using the pre-trained model for their own purposes may run into the following roadblocks:

- [x] Making sure data is in wide format for keras 
- [x] Windowing their input signal as described in the manuscript
- [x] Extracting features correctly
- [x] Organizing input features (3d array)
- [x] Normalizing signals to min/max based on manuscript data
- [x] Filtering and general paper method replication
